### PR TITLE
fix: dispute state after arbitration

### PR DESCRIPTION
**Pull request by richard-ramos**

### CODE REVIEW

File: src/js/features/arbitration/reducer.js [hunk 1]
```diff
@@ -83,13 +83,26 @@ function reducer(state = DEFAULT_STATE, action) {
...
+      const currentEscrow = escrows.find(x => x.escrowId === action.escrowId);
+      if(currentEscrow) {
+        currentEscrow.arbitration.open = false;
+        currentEscrow.arbitration.result = action.result;
+      }
+
       return {
         ...state, ...{
           escrows,
+          arbitration,
```
Comment (jrainville) [inline src/js/features/arbitration/reducer.js]: To not modify the state directly:
```suggestion
          arbitration: {
            ...state.arbitration,
            arbitration: {
              arbitration: {
                open: false,
                result:  action.result
              }
            }
          },
```
You can then get rid of the code above

File: src/js/features/arbitration/reducer.js [hunk 2]
```diff
@@ -83,13 +83,26 @@ function reducer(state = DEFAULT_STATE, action) {
...
+    {
+      const arbitration = state.arbitration;
+      arbitration.arbitration.open = false;
+      arbitration.arbitration.result = action.result;
+
+      const currentEscrow = escrows.find(x => x.escrowId === action.escrowId);
+      if(currentEscrow) {
+        currentEscrow.arbitration.open = false;
+        currentEscrow.arbitration.result = action.result;
+      }
```
Comment (jrainville) [inline src/js/features/arbitration/reducer.js]: This modifies the state directly and also, I just noticed that we have escrows inside the arbitration reducer. Does it make sense?
Do we duplicate the escrows in two reducers or in this one it's only about the arbitrations in escrows?

File: src/js/features/arbitration/reducer.js (hunk 1 above)
Comment (richard-ramos) [reply src/js/features/arbitration/reducer.js]: ahh! I always forget this. Sorry!

File: src/js/features/arbitration/reducer.js (hunk 2 above)
Comment (richard-ramos) [reply src/js/features/arbitration/reducer.js]: It's only arbitrations in escrows!

**Pull request merged**